calico/reference/configure-cni-plugins.mdx

```diff
@@ -248,6 +248,35 @@ you must also run calico/kube-controllers with the policy, profile, and workload
 
 When using `type: k8s`, the {{prodname}} CNI plugin requires read-only Kubernetes API access to the `Pods` resource in all namespaces.
 
+### Enabling Policy Setup Timeout
+
+If you wish for the CNI to delay a new pod from starting until the pod's policy has been programmed to the dataplane,
```

Comment:
Some new copy, based partly on the previous sections and incorporating the idea of a note you have below:

```
The `policy_setup_timeout_seconds` option makes the CNI plugin wait to start a new pod until one of the following conditions occurs:

* The pod's policy has finished being programmed.
* A specified amount of time has elapsed.

By setting this option, you can avoid errors that can occur when a pod tries to start before the pod's policy is programmed.

Example CNI config:
```